packages/terra-clinical-data-grid/tests/jest/__snapshots__/DataGrid.test.jsx.snap

```diff
@@ -3369,10 +3369,10 @@ exports[`getDerivedStateFromProps correctly applies the theme context className
             className="main"
           >
             <Scroll
-              className="normalizer"
+              className="normalizer orion-fusion-theme"
```

Comment:
I am actually not sure why this needs to be updated, but apparently we were never injecting our themes into the datagrid component from lerna before this? Upgrading to lerna v6 apparently requires us to do this.

Reply:
This is not related to lerna, it's due to a dependency generating updated html (likely terra-content-container in this case)